### DISCUSSION TITLE
Add Rumble to Your List of Video Sites

### DISCUSSION THREAD

**Discussion by moshemo613** [FrostCo/AdvancedProfanityFilter#562] [Ideas]
I would like to make a feature request; namely, that you add Rumble to your list of sites that you can mute audio on.

**Comment by richardfrost** [comment]
Thanks for the suggestion @moshemo613!

Can you give this config a try?

```json
{
  "rumble.com": [
    {
      "mode": "cue",
      "videoSelector": "div.media-container div.video-player video"
    }
  ]
}
```

To use this config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

Unfortunately it looks like Rumble doesn't have any auto-captions like YouTube, so it will only work on a video that actually includes captions. I was able to find some to test with by searching for `closed captions`.

**Comment by richardfrost** [comment]
This has now been included in version [3.5.1](https://github.com/FrostCo/AdvancedProfanityFilter/releases/tag/3.5.1). Let me know how it works for you @moshemo613. Thanks again for the suggestion!
